app/models/inactive_contacts.rb

```diff
@@ -7,15 +7,9 @@ def initialize(contacts = Contact.archivable)
 
   def archive(verified: false)
     contacts.each do |contact|
-      log("Archiving contact: id(#{contact.id}), code(#{contact.code})")
-
+      puts "Archiving contact: id(#{contact.id}), code(#{contact.code})"
```

Comment:
Is there any reason we are not using logger?

Reply:
Logging is currently work in progress. Guess I asked for review too early 🙂